src/common/perf_counters.h

```diff
@@ -351,6 +353,7 @@ class PerfCountersCollectionImpl
 
 private:
   void dump_formatted_generic(ceph::Formatter *f, bool schema, bool histograms,
+                              bool dump_labeled = false,
```

Comment:
I'd avoid defaulting `dump_labeled` here and everywhere else.  The default of false is kind of backwards because going forward we want labeled counters to be dumped everywhere -- `perf dump` behavior is kept around just for backwards compatibility.  Flipping the default to true isn't particularly attractive either because, unlike `logger` and `counter` which are just filters, `dump_labeled` also affects the format of the output -- in this regard it's much closer to non-defaulted `schema` than to defaulted `logger` and `counter`.

Reply:
This got marked resolved but there is still a default in the code here.